rpi-eeprom-update

```diff
@@ -189,6 +189,30 @@ prepareImage()
    fi
 }
 
+runFlashrom()
+{
+   retries=5
+   freq=16000
+
+   # If flashrom fails then retry at a lower speed.
+   while [ ${retries} -gt 0 ]; do
+      echo "flashrom -p linux_spi:dev=${SPIDEV},spispeed=${freq} -w ${BOOTFS}/pieeprom.upd"
+      if flashrom -p linux_spi:dev=${SPIDEV},spispeed=${freq} -w "${BOOTFS}/pieeprom.upd" >> "${FLASHROM_LOG}"; then
+         echo "Verifying update"
+         if flashrom -p linux_spi:dev=${SPIDEV},spispeed=${freq} -v "${BOOTFS}/pieeprom.upd" >> "${FLASHROM_LOG}"; then
+            echo "VERIFY: SUCCESS"
+            return 0
+         else
+            echo "VERIFY: FAIL"
+         fi
+      fi
+      echo "Failed to update bootloader SPI flash using flashrom. Retrying."
```

Comment:
Do we want to avoid printing this message on the last iteration?

Reply:
Yes will change